### ISSUE TITLE
Quickstart project is not working

### ISSUE THREAD

**Issue by trushalip2** [angular/angular#19172]
I have downloaded the Quickstart project from angular.io website. But even after installing dependencies it is giving me errors like Cannot find module '@angular/core'
![quickstarterror](https://user-images.githubusercontent.com/30066930/30370026-eec4bee6-9893-11e7-8c93-0f07ad477d9a.png)

**Comment by gkalpak** [comment]
Verified :shipit: 

@Foxandxss any idea what might be going one here? Is this going to be reworked as part of moving the examples to the cli?

**Comment by Foxandxss** [comment]
Uh, ah, what the hell. That is quite the error indeed.

The repo is probably going to be deprecated as soon as we land the CLI stuff. But still, worth a look.

**Comment by EvanCarroll** [comment]
@Foxandxss Do you have a patch to upgrade tour of heroes to CLI? I could submit one.

**Comment by Foxandxss** [comment]
I appreciate the intention @EvanCarroll but it is not a simple thing. Upgrade the ToH to CLI is at least a 8 hours work. And well, it is underway already. But before we can land that we need to land a big internal rework that I am about to land this week.

**Comment by EvanCarroll** [comment]
Be glad to help out if it kills it off faster. I don't think the patch to kill quickstart would be that bad, but I have no idea what you're working up. Really, the majority of it is in the Setup. I think for the purposes of Quickstart, after we install the CLI we just need `ng new --minimal toh;`.

From there, the rest is an upgrade. I would think it would be better to rely on the CLI rather than text creation. But, the upgrade isn't needed. I could provide help there too, but seeding the project with the CLI will get people up and going where as now things are mostly broke.

**Comment by Foxandxss** [comment]
We are working on a very very big project right now to migrate all documentation to CLI. That includes everything that happens behind the scenes (which is systemjs currently), e2e testing, unit testing, plunkers, zips. Once that is done (I want to create a PR For that today) we will revamp **all** docs for the CLI.

ToH is not that easy as removing the setup part and say `new new toh`. Internally it is 6 different folder examples (so the work needs to be done 6 times). Components needs to be moved to their own folders, change some prefixes, change the prose to not say `Create a component named xxx` but `Generate a component using ng generate....` Needs to explain a few thing that are CLI related, remove all comments of SystemJS. We also plan to remove some explanations that are not really helping, and a few more things.

So again, I really appreciate the help and we are sorry this is taking long, but most of the times the changes are not as easy as we think they are.

**Comment by trushalip2** [comment]
I am talking about project which is downloaded from  **https://angular.io/guide/quickstart** . That downloaded project is not executing.

**Comment by EvanCarroll** [comment]
@trushalip2 are you trying to learn angular with  Tutorial: Tour of Heroes? Or are you starting a rl project?

**Comment by Foxandxss** [comment]
The error is not  happening to me.

**Comment by trushalip2** [comment]
@EvanCarroll  @Foxandxss  I am not trying Tour of hero..It is also working for me.I am trying hello world of CLI quickstart project...without using CLI tool....But just downloading that Hello world program from https://angular.io/guide/quickstart

**Comment by Foxandxss** [comment]
Ah, that quickstart. Did you download the zip from there?

**Comment by glalwani2** [comment]
I am getting the same error as @trushalip2 . I just downloaded the quickstart seed code and did npm install.On doing npm start I am getting the above error.Strange that seed project is not compiling and this bug is still open.Please help!

**Comment by gkalpak** [comment]
We are moving away from that quickstart and towards `@angular/cli`-based projects. All the examples in docs are in the process of being converted to cli-based projects. I suggest you use `@angular/cli` too. 

(Once you have installed it globally (with `npm install -g @angular/cli`), scaffolding a new app is as simple as `ng new MY_PROJECT_NAME`.)

**Comment by EvanCarroll** [comment]
What's wrong with the doc team? Updating these docs is a three hour feat. It's going on 3 months [since this was the preferred method](https://stackoverflow.com/questions/45422865/angular-cleaning-up-build-files#comment77810905_45424599). I volunteered to do or finish the work. Are you looking for assistance?

@Foxandxss said,

> And well, it is underway already. But before we can land that we need to land a big internal rework that I am about to land this week.

Where is the information about this and on what branch or where is this work being done?

**Comment by Foxandxss** [comment]
Please refrain of using those comments.

It is not a 3 hours feat as you say, we are over 100h working on the docs. They will be out this week.

**Comment by kylecordes** [comment]
I happened across this today, and noticed that the old QuickStart is clearly marked deprecated (of which I heartily approve! The new CLI approach is great!). But I happened across that because it is still linked from the documentation. Maybe just unlinking it from the documentation would be enough to have this item closed and abandoned.

**Comment by jenniferfell** [comment]
docsarea=quickstart

**Comment by jenniferfell** [comment]
Keeping open to recheck. 
README.md for cli-quickstart.zip that I downloaded today says: "This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.0.0-rc.0."

But the package.json looks like this has been updated to v6.

**Comment by brandonroberts** [comment]
Closing this one as the examples are working correctly now. Opened a [new issue](https://github.com/angular/angular/issues/23936) to track removal of outdated README that's provided with the zip file.
